scripts/generate-app-config/generateAppConfig.js

```diff
@@ -28,7 +28,9 @@ const addConfig = (config, fileName, buildPath, fs, imports) => {
 /**
 * Writes out a file consisting of the app config and imports with the given file name to the specified path.
 */
-const generateAppConfig = (siteConfig, production, verbose, locale) => {
+const generateAppConfig = ({
+  siteConfig, mode, prefix, apps = [], verbose = false, basename,
+}) => {
```

Comment:
Can we provided the apps default value through the default site configuration like we do for extensions, etc? https://github.com/cerner/terra-dev-site/blob/e286778fcbadb137d7d95ff61a264cdcc4133850/config/site/site.config.js#L105

Reply:
I could, but I don't want to expose apps in the config api.